go/test/endtoend/vtgate/queries/random/schema.sql

```diff
@@ -0,0 +1,20 @@
+CREATE TABLE emp (
```

Comment:
not for this PR, but as follow up work. I think we need more complex schema here. We are missing important types, such as float, date and json.